sdk/data/aztables/README.md

```diff
@@ -1,151 +1,343 @@
-# Azure Tables client library for Go
+# Azure Data Tables client library for Go
 
-Azure Table storage is a service that stores large amounts of structured NoSQL data in the cloud, providing
-a key/attribute store with a schema-less design.
+Azure Data Tables is a NoSQL data storage service that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS.
+Tables scales as needed to support the amount of data inserted, and allow for the storing of data with non-complex accessing.
```

Comment:
```suggestion
Data Tables scales as needed to support the amount of data inserted, and allow for the storing of data with non-complex accessing.
```
(or maybe not?)

Reply:
(also side-note - what's an example of non-complex accessing? Is it because it feels more key/value-ish?)